### ISSUE TITLE
pypi powerutils package is not this project

### ISSUE THREAD

**Issue by Riesi** [embedded-machine-learning/powerutils#5]
Trying to install the package of this repository via pip doesn't work, since the package on the pypi website is a different project.
https://pypi.org/project/powerutils/#description

**Comment by mwessley** [comment]
Correct. The naming is unlucky since a pypi project with the same name already exists.
Local install via git clone and `pip3 install -e .` is the current workaround.
